.github/workflows/bedrock-pull.yaml

```diff
@@ -3,6 +3,8 @@ on:
   pull_request:
     paths:
       - bedrock/**
+    paths-ignore:
+      - bedrock/package-lock.json
```

Comment:
why the ignore? if the lockfile changed should the pr check to see that the new versions in the lockfile work?

Reply:
My motivation was that a change to the lockfile shouldn't result in a new attempted release of a library (since it doesn't get included in the tarball). But I guess it's okay if it still results in a PR build - I'll change it

Reply:
Even though a lockfile doesn't get included in the tgz, a change in a lockfile can still result in a change to the final build artifact. Both transitive deps and loosely pinned deps can change without a change to the package.json.

Reply:
Huh, really? That's interesting/surprising to me. I imagined that would only apply if the dependencies were bundled?

Reply:
It's for the build/test tools rather than the runtime dependencies. It's like the node, npm, or linux version used to build the package. If those change it can technically change the resulting artifact. or different transitive testing dependencies can cause tests to fail.